specificatie/openapi.yaml

```diff
@@ -557,6 +612,78 @@ components:
         pattern: '^[0-9]{16}$'
         example: '0226010000038820'
 
+    postcode:
+      description: Postcode van het adres.
+      name: postcode
+      in: query
+      required: false
+      schema:
+        type: string
+        pattern: '^[1-9]{1}[0-9]{3}[a-zA-Z]{2}$'
+        example: '6922KZ'
+
+    huisnummer:
```

Comment:
Volgens mij hebben we afgesproken dat query-parameters de post-fix "-query" in de naam krijgen om onderscheid te kunnen maken met path-parameters op dezefde property. Dus huisnummer-query.